portfolio.html

```diff
@@ -4,8 +4,42 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
+    />
     <link rel="stylesheet" href="style.css" />
     <title>Portfolio</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+      <div class="brand">
+        <h1 class="brand-name">Brand Name</h1>
+      </div>
```

Comment:
This should be an `<a>` tag that says `My Logo`.

<img width="444" alt="Screen Shot 2022-08-18 at 8 37 50" src="https://user-images.githubusercontent.com/82426682/185408861-77ad858d-f044-4a69-ab69-d1d0aaf6fd1e.png">

Please fix it.